lib/controllers/posts.js

```diff
@@ -0,0 +1,33 @@
+const { Router } = require('express');
+const authenticate = require('../middleware/authenticate');
+const authorize = require('../middleware/authorize');
+const Post = require('../models/Post');
+
+module.exports = Router()
+  .get('/', authenticate, authorize, async (req, res, next) => {
+    try {
+      const posts = [
```

Comment:
This was in the demo code but it limits this get route to only ever receiving this baked in post and not whatever is in the database.

Reply:
You need a Post.getAll